src/safeds_stubgen/api_analyzer/_api.py

```diff
@@ -252,9 +261,11 @@ class Parameter:
     name: str
     is_optional: bool
     default_value: str | bool | int | float | None
+    default_is_none: bool
     assigned_by: ParameterAssignment
     docstring: ParameterDocstring
-    type: AbstractType
+    type: AbstractType | None
+    is_type_inferred: bool
```

Comment:
Can we store `is_type_inferred` (renamed to `is_inferred`) in `AbstractType`?

Reply:
@lars-reimann While moving it to the AbstractType I noticed that we don't even use that field for anything. Do you think we will need it in the future or can I remove it completely?

Reply:
It indicates that we are not be entirely certain about the type (compared to manifest types). However, if it's not used at the moment, remove it.